includes/custom-measurements-units-disclaimer.mdx

```diff
@@ -0,0 +1,5 @@
+<Note>
+
+Currently, unit conversion is only supported once the data has already been stored. This means that, for example, `('myMetric', 60, 'second')` and `('myMetric', 3, 'minute')` would not be aggregated, but rather stored as two separate measurements. To avoid this, make sure to use a consistent unit when recording a custom measurement.
```

Comment:
Should the names rather be myMeasurement corresponding with the rename?

Reply:
good catch!